diffkemp/simpll/DifferentialFunctionComparator.h

```diff
@@ -39,6 +44,9 @@ class DifferentialFunctionComparator : public FunctionComparator {
     /// Specific comparison of attribute lists.
     /// Attributes that do not affect the semantics of functions are removed.
     int cmpAttrs(const AttributeList L, const AttributeList R) const override;
+
+  private:
+    const ElementIndexToNameMap *EINM;
```

Comment:
I think it would be better to have a pointer to DebugInfo here. It'll be easier when we need some additional debug information in the FunctionComparator in future.